pyuvsim/simsetup.py

```diff
@@ -851,7 +855,8 @@ def parse_time_params(time_params):
                 raise ValueError("Either integration_time or Ntimes must be "
                                  "included in parameters: " + kws_used)
             if st and et:
-                time_params['duration'] = time_params['end_time'] - time_params['start_time'] + time_params['integration_time'] * dayspersec
+                time_params['duration'] = time_params['end_time'] - time_params['start_time'] + time_params[
+                    'integration_time'] * dayspersec
```

Comment:
same as line-breaking comment

Reply:
Fixed in b7e89d70fab47c00273dbfa95fa5f5c82f3af8b0